semeio/workflows/csv_export2/csv_export2.py

```diff
@@ -42,6 +42,9 @@
 requested in the Eclipse DATA file. A wildcard like ``W*`` can in certain cases
 (e.g. Eclipse simulations with 100+ wells) produce thousands of vectors, and can
 then be replaced by something more explicit like ``WOPT* WGPT* WWPT*``.
+
+Dates beyond year 2262 are not supported. Use ecl2csv in combination with
+merge_csv in such scenarios.
```

Comment:
Would it be possible to add an example of this use?

Reply:
I think it is covered by the examples provided by ecl2df and subscript (as forward models and workflows injected into the ert documentation in fmu-docs.